### DISCUSSION TITLE
Microscopy Slide for Multiplexing I VIP-testers wanted

### DISCUSSION THREAD

**Discussion by Tina-ibidi** [IBEXImagingCommunity/ibex_imaging_knowledge_base#235] [Show and tell]
Hi all, I am a product manager at ibidi and we have developed a new slide for multiplexing that can be combined with flow systems. We are currently looking for VIP-testers to receive feedback on the product and gain some insights on multiplexing workflows and applications. If you are interested, please get in touch 
![sticky_Slide_Tissue](https://github.com/user-attachments/assets/d782aafd-3587-4427-acbc-d929f633c6c4)

**Comment by radtkea** [comment]
Thank you so much for sharing Tina!

Could you share a few more details about the chamber with us? 

1. For example, does one have to image through a glass slide? Or could you flip the slide over and image through a coverslip?
2. What objectives are compatible with the chamber? It looks like you are using a 10X in the image. Is this a limitation?
3. What is the volume of fluid included in the chamber? Looks like 167 ul?

Check this out: @colinchu72, @KoenigJFE, @nzangger, and @eliisakek. This looks very [promising](https://ibidi.com/sticky-slides/322-sticky-slide-tissue.html).

Best wishes,
A

**Reply by rvmoriarty** [reply]
Keep me in the loop!

**Reply by Tina-ibidi** [reply]
Hi Andrea, 

thank you very much for your message and questions!
Please find my answers bellow:

1.	Currently, the sticky-Slide Tissue is designed for use with standard 1mm-thick glass slides. Flipping the slide over isn't feasible due to the Luer ports being in the way. However, after staining, you could remove the sticky-Slide Tissue, add a 1.5 coverslip, and then flip it upside down if needed.

2.	The objectives used are constrained by their working distance and the glass slide thickness used with the sticky-Slide Tissue. So far, the sticky-Slide Tissue is intended for use with standard glass slides (26 x 76 x 1 mm), and we have tested it primarily with a 10x objective. However, we're exploring the potential demand for 1.5H coverslips with a length that is compatible with the sticky-Slide Tissue dimensions (commercially avaiable glass coverslips are normally too long for the sticky-Slide Tissue), which would enable the sticky-Slide Tissue to be used in high-resolution or super-resolution applications. Would this also be essential for your multiplexing applications?

3. The chamber volume is 165 µl.

I hope this helps! Please reach out if you have more questions and please let me know what you think of the product, for example if there are any features missing in your opinion etc. We are very eager to learn from the experts!

Best,
Tina

**Reply by radtkea** [reply]
Thanks Tina! Happy to see the interest and support from everyone here in this thread! Thanks @wolheu, @DasEFX130, and @eallen1738. Best of luck with everyone's experiments!

**Comment by wolheu** [comment]
Hi Tina,

Thanks for sharing the information. We have used the sticky-slide tissue in the last weeks on standard glass slides with LD objectives but also with cryo sections on  24x50mm cover slips with high NA objectives (20x 0,8Na looking for bacteria distribution on organ sections).
The coverslip needs to be positioned perfectly in the middle to be tight at the entrance and exit. However, it works for us nicely.
Concerning the volume, we have use the Aria with M-switch and use a AB mix (primary label ABs) with a total volume of 450-600ul for each round. This includes volume of the chamber, all tubing and preloading. 

Best regards,
Wolf

**Reply by Tina-ibidi** [reply]
Hi Wolf, 

Thank you for sharing this additional information and tips! Glad to hear that you successfully used the sticky-Slide Tissue on thin glass coverslips in combination with a flow system.

Best,
Tina

**Reply by DasEFX130** [reply]
Hi Tina,
Question (and apologies for the ignorance): what kind of flow systems work with it? Thanks so much!

**Reply by Tina-ibidi** [reply]
Thank you for your question! In principle, the sticky-Slide Tissue can be used with any flow system (syringe pumps, air pressure pumps, peristaltic pumps), provided that the tubing has a Luer connector compatible with the slide's Luer port.

**Comment by eallen1738** [comment]
Hi Tina, we are very interested. We would want to use a 1.5 coverslip instead of a slide with it. Would this be possible?

**Reply by wolheu** [reply]
Hello,
We have used the sticky - slide tissue with different 1.5 coverslip by now. It does work fine in our hands. One way to make them fit to the size of the sticky slide tissue and therfor more robust in the stage insert is to cut the overhang with a Diamant knife/pen. We had very little loss so far.  A couple cryo sections samples ended up with a crack and where lost.  However the image quality is way better due to high NA  objectives.
Best regards,
Wolf

**Reply by Tina-ibidi** [reply]
> Hi Tina, we are very interested. We would want to use a 1.5 coverslip instead of a slide with it. Would this be possible?

Hi, 
yes, using the slide with a 1.5 coverslip is possible! We are planning to add coverslips adjusted to the length of the sticky-Slide Tissue to our standard product portfolio soon. If you are interested in becoming a VIP-Tester, please write to info@ibidi.de with the comment "please forward to Tina"; we can then discuss details over Email.
Best,
Tina
